dianna/dashboard/Home.py

```diff
@@ -38,18 +38,55 @@
 
 # Display the content of the selected page
 if selected == "Home":
```

Comment:
I was looking at the layout on a wide screen, and think it might look nicer if the text is in a centered column, so as to have a bit of whitespace on the sides when the screen is very wide

Reply:
Agree.

Reply:
@laurasootes for some reason I cannot commit your suggestion. Getting error: "Applying suggestions on deleted lines is currently not supported." Will try to spot the difference and edit myself.